cmd/daserver/daserver.go

```diff
@@ -149,12 +149,15 @@ func startup() error {
 	log.Root().SetHandler(glogger)
 
 	if serverConfig.Metrics {
+		if len(serverConfig.MetricsServer.Addr) == 0 {
+			fmt.Printf("Metrics is enabled, but missing --metrics-server.addr")
+			return nil
```

Comment:
Shouldn't we return an error here? Or is the idea that returning nil exits the node early with "Metrics is enabled, but..." being the last printout?

Reply:
Exactly, make it easier to see the actual error instead of buried in a stack trace and such

Reply:
The config value defaults to 127.0.0.1 so unless it's being manually cleared, I don't think this would hit? I also thought we errored if the json config contained keys that aren't valid. It might be worth doing that to avoid general bug risk from config typos like in the CLI